src/test/java/com/coreos/jetcd/internal/impl/ClientBuilderTest.java

```diff
@@ -37,4 +38,37 @@ public void testEndPoints_Verify_SomeEmpty() {
   public void testBuild_WithoutEndpoints() {
     builder.build();
   }
+
+  @Test(dataProvider = "validEndpoints")
+  public void testBuild_WithValidFormatEndpoints(String endpoint) {
+    builder.setEndpoints(endpoint);
+  }
+
+  @DataProvider
+  private Object[][] validEndpoints() {
+    return new Object[][] {
+        {"localhost:2379"},
+        {"127.0.0.1:2379"},
+        {"fe80::6e40:8ff:fe:2380"},
```

Comment:
don't let `"localhost:2379"`, `"127.0.0.1:2379"`, and `"fe80::6e40:8ff:fe:2380"` to be valid endpoint since they don't specify a scheme.